src/website/views/components/header.pug

```diff
@@ -827,6 +827,157 @@ h3 Header with Tabs, Icon Buttons, and Avatars
     <script>chi.drawer(document.getElementById('drawer-trigger-1'));</script>
     <script>chi.drawer(document.getElementById('drawer-trigger-4'));</script>
 
+h3 Inverse Header with Colorbar divider and horizontal Tabs navigation
```

Comment:
Let's change to: Enterprise Portal Header with Horizontal Navigation

Reply:
All done! Thanks! I also added a backstop test case for this. I think in the future we should add responsive tests for each example of Header since it changes in different viewports.